src/test/java/teammates/test/cases/browsertests/InstructorSearchPageUiTest.java

```diff
@@ -128,7 +127,7 @@ private void testDeleteAction() throws Exception {
         searchPage.clickDeleteAndCancel(courseId, studentName);
         assertNotNull(BackDoor.getStudent(courseId, studentEmail));
 
-        String expectedStatus = "The student has been removed from the course";
+        String expectedStatus = Const.StatusMessages.STUDENT_DELETED;
         searchPage.clickDeleteAndConfirm(courseId, studentName);
         InstructorCourseDetailsPage courseDetailsPage = searchPage.changePageType(InstructorCourseDetailsPage.class);
         courseDetailsPage.waitForTextsForAllStatusMessagesToUserEquals(expectedStatus);
```

Comment:
Can just use `Const.StatusMessages.STUDENT_DELETED` and get rid of `expectedStatus`.